data/Glitched World/Bottom of the Well MQ.json

```diff
@@ -2,28 +2,136 @@
+            "Bottom of the Well MQ Basement ReDead 2": "can_kill_redead",
+            "Bottom of the Well MQ Basement ReDead 3": "can_kill_redead",
+            "Bottom of the Well MQ Basement ReDead 4": "can_kill_redead",
+            "Bottom of the Well MQ Basement ReDead 5": "can_kill_redead",
+            "Bottom of the Well MQ Coffin Room Gibdo": "
+                (Small_Key_Bottom_of_the_Well, 2) and can_kill_redead",
+            "Bottom of the Well MQ False Floor Wallmaster": "
+                can_kill_wallmaster and (is_adult or Sticks or has_explosives or Slingshot or Boomerang)",
+            "Bottom of the Well MQ False Floor Floormaster": "
+                can_kill_floormaster and (is_adult or Sticks or has_explosives or Slingshot or Boomerang)",
+            "Bottom of the Well MQ Inside Cage Skullwalltula Lower": "
+                has_soul(Skullwalltula) and
+                (can_use(Dins_Fire) or
+                    ((Small_Key_Bottom_of_the_Well, 2) and
+                        (is_adult or Sticks or has_explosives or Slingshot or Boomerang)))",
+            "Bottom of the Well MQ Inside Cage Skullwalltula Higher": "
+                has_soul(Skullwalltula) and
+                (can_use(Dins_Fire) or
+                    ((Small_Key_Bottom_of_the_Well, 2) and
+                        (is_adult or Sticks or has_explosives or Slingshot or Boomerang)))",
+            "Bottom of the Well MQ Dead Hand": "
+                has_soul(Dead_hand) and (is_adult or Kokiri_Sword or (Sticks and logic_child_deadhand))",
+            "Bottom of the Well MQ GS Basement": "can_child_attack or is_adult",
             "Bottom of the Well MQ GS Coffin Room": "
-                can_child_attack and
-                (Small_Key_Bottom_of_the_Well, 2)"
+                (Small_Key_Bottom_of_the_Well, 2) and (can_child_attack or is_adult)",
+            "Wall Fairy": "has_bottle and can_use(Slingshot)" # The fairy pot is obsolete
         },
         "exits": {
-            "Kakariko Village": "True"
+            "Bottom of the Well Middle": "
+                can_play(Zeldas_Lullaby) or (logic_botw_mq_pits and has_explosives)
+                    or (can_weirdshot and can_use(Longshot))"
+        },
+        "events": {
+            "Bottom of the Well West Switch Pressed": "((can_pass_boulder('BOTW_LEFT_SIDE_BOULDER_1') or can_pass_boulder('BOTW_LEFT_SIDE_BOULDER_2')) and can_child_attack or Nuts) or (can_play(Zeldas_Lullaby) and logic_botw_mq_pits)
+                or (can_weirdshot and can_use(Longshot))"
+        }
+    },
+    {
+        "region_name": "Bottom of the Well Middle",
+        "dungeon": "Bottom of the Well",
+        "locations": {
+            "Bottom of the Well MQ Map Chest": "True",
+            "Bottom of the Well MQ East Inner Room Freestanding Key": "True",
+            "Bottom of the Well MQ Lens of Truth Chest": "
+                (Small_Key_Bottom_of_the_Well, 2) and has_explosives",
```

Comment:
Adult can antigrav there with just explosives. I haven't been able to find a setup for child.

Reply:
There's probably a video out there, but I just recorded a new one https://youtu.be/kw8IWMNg_ZE?si=WOLqqStBPgXxQUZd

Reply:
It occurred to me that I don't know how to write the instructions for a left megasidehop so feel free to comment if the description is bad.